src/gui/tray/SyncStatus.qml

```diff
@@ -0,0 +1,110 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import Style 1.0
+
+import com.nextcloud.desktopclient 1.0 as NC
+
+RowLayout {
+    id: layout
+
+    property alias model: syncStatusModel
+
+    spacing: 0
+
+    NC.SyncStatusModel {
+        id: syncStatusModel
+    }
+
+    Image {
+        id: syncIcon
+
+        Layout.alignment: Qt.AlignLeft | Qt.AlignVCenter
+        Layout.topMargin: 16
+        Layout.bottomMargin: 16
+        Layout.leftMargin: 16
+
+        source: syncStatusModel.syncIcon
+        sourceSize.width: 32
+        sourceSize.height: 32
+    }
+
+    Item {
```

Comment:
please merge both `ColumnLayout` into a single layout and remove the parent `Item` that should not be needed